perl_modules/EBI/FGPT/Resource/Database/pgGXA.pm

```diff
@@ -351,7 +350,7 @@ sub fetch_baseline_experiment_info_from_atlasdb {
     my ( $self, $logger ) = @_;
 
     my $query = "
-        select accession, last_update, title from experiment
+        select accession, last_update from experiment
```

Comment:
ahaha, I remember these. See, keeping to sources was going to be a pain. Happily this PR will avoid this in the future!